components/authentication/base/build.yaml

```diff
@@ -86,43 +86,6 @@ roleRef:
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: component-maintainers
-  namespace: build-service
-subjects:
-  - kind: User
-    apiGroup: rbac.authorization.k8s.io
-    name: sbose78
-  - kind: User
-    apiGroup: rbac.authorization.k8s.io
-    name: Michkov
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: component-maintainer
----
-kind: RoleBinding
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: component-maintainers
-  namespace: jvm-build-service
-subjects:
-  - kind: User
-    apiGroup: rbac.authorization.k8s.io
-    name: stuartwdouglas
```

Comment:
Please add stuart to jvm-build-maintainer.yaml, he is removed here but not added there.

Reply:
added 

```YAML
kind: RoleBinding
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: jvm-build-maintainers
  namespace: jvm-build-service
subjects:
  - kind: User
    apiGroup: rbac.authorization.k8s.io
    name: stuartwdouglas
  - kind: User
    apiGroup: rbac.authorization.k8s.io
    name: gabemontero
  - kind: User
    apiGroup: rbac.authorization.k8s.io
    name: Michkov
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: jvm-build-maintainer
```